### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dcndr**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +22,6 @@ Use this repository to self-host Assembly with the source code.
 
     Run `docker pull ghcr.io/SubjectiveLabs/subjective-assembly:latest` to pull the image.
 
-3. Connect to the server with Subjective.
-    If your school is public on Subjective Discovery, changes to your school'll be reflected to all students of your school on the next cache clear.
-    If not, Assembly's share link can be input in Subjective to retrieve your school's bell time data.
+3. Configure your school's bell times, notices, and links by opening Assembly (if you are accessing it directly on your server, it is most likely localhost/app), otherwise it will be *[server-address]*/app
+
+4. Connect to the server with Subjective by inputting the host link of Assembly, this would be the server's local IP address, or if the server is on the web, it's web address.
```
Comment (circles-png) [inline README.md:27]: inputting is not a very good word
the first comma doesnt make sense
`it's` is wrong

**Review by circles-png — CHANGES_REQUESTED** [review]
write some stuff about the api endpoints here or in the wiki

File: README.md (hunk 1 above)
Comment (dcndr) [reply README.md:27]: this is why im failing english

Comment (circles-png) [reply README.md]: couldnt be me

**Review by circles-png — APPROVED** [review]

**Pull request merged**